### PR TITLE
Transform to REST based on SOAP version of the plugin

### PR DESCRIPTION
**Pull request by gonzunigad**
As we made a lot of improvements on the SOAP plugin since this plugin was stalled, we decided to create a new version of this REST plugin based on the latest version of the SOAP plugin.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+script: echo "We don't have tests yet :("
+
+before_deploy:
+- sh package.sh
+
+deploy:
+  provider: releases
+  name: "$TRAVIS_TAG"
+  api_key:
+    secure: erCD8ZJCL0GJ06K/YwtDXBcbQRgA/Ol+Um5ebPwBk3Cm+Q0UqNm5lssqg88MhhW0Pg5UIO7rzZg/36UX43nlRew/MbJLSYV+dxLNHjNsSxuxx1a3rEMw7YH26Y1FekC7o4jZ4H0I5mSlKobFlshbwIH57WkMrTDYYuJVbuCshQH+FxxqCgFcJo9bvcr48pTc4h4MWyk5IBlNeHqyvSDuypku9A9Eo9PpTJLQPqtwPcI0he9vY6z6QMs3pOyHJ5t6lLoL9Ks0Q8ea+YRw1xK4kbXK4LrwBx+owstB0VwP2XH63onPJC8fser8FxNHq3W4jL2rqhHt1K9Wi0Psc6ST17yuDdlRs18QWfWqKsVjxeTvAwv0W2whBoKrRgmYrdFzjcutvzA76pphOoC0AISe9Ji/U8b+2g6EDlgLkCl5IoYw/nUGbwt5VlrrRs5uAaexBXoMXynM8tUSCKXDsdmSr3/I4Bq/ScMx1gfdvTx5q9tBGVWhT6LaW3iFTdDHLmm0yUsFgjK0ks5YsXF1jBwQMvjVYKxJncwCmJVh291jJ4WsBwV3EWfSYxaw+Z3yrzf8jRvPR84O+uEbbMJYD63G6fJ9gxMSXSI+XSrvBy9f6TPymoNYj+8vKdN0X/kcY8jXpVwEf4OWXr4lL8qvpiFr+ZOn7yz8V9+WH1pMITluJlg=
```
Comment (ffiebig) [inline .travis.yml:15]: Will this point to the SOAP version and overwrite it ?

File: plugin/libwebpay/ConnectionCheck.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+class ConnectionCheck {
+    public static function check()
+    {
+
+        require_once(__DIR__  . '/ConfigProvider.php');
+
+        $configProvider = new ConfigProvider();
+        $config = array(
+            'MODO' => $configProvider->getConfig('webpay_rest_test_mode'),
+            'COMMERCE_CODE' => $configProvider->getConfig('webpay_rest_commerce_code'),
```
Comment (ffiebig) [inline plugin/libwebpay/ConnectionCheck.php:14]: This should always point to integration, so it doesn't fail when in production

File: plugin/libwebpay/certificates.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+/**
+ * Codigo Comercio
+ **/
+"commerce_code" => "597020000540",
+
+/**
+ * Llave Privada
+ **/
+"private_key" => "-----BEGIN RSA PRIVATE KEY-----
```
Comment (ffiebig) [inline plugin/libwebpay/certificates.php:19]: Should this be here ?

**Comment by ffiebig** [comment]
There a few .DS_store files, it wouldn't be bad to add them to .gitignore

File: plugin/libwebpay/ConnectionCheck.php (hunk 2 above)
Comment (gonzunigad) [reply plugin/libwebpay/ConnectionCheck.php:14]: It is actually working on this way, so you can check if your new credentials works.  :)

**Comment by gonzunigad** [comment]
> There a few .DS_store files, it wouldn't be bad to add them to .gitignore

Actually, i am deleting them. It is on the .gitignore. The original plugin has the .DS_Store files.

**Pull request merged**